lib/ui.rb

```diff
@@ -1,31 +1,28 @@
 require_relative 'messages'
 
 class UserInterface
-    def initialize
-        @message = Message.new
-    end
 
-    def start_game
-        puts @message.lookup(:welcome)
-        puts @message.lookup(:instructions)
-        puts display_board
+    def display_board(board)
```

Comment:
I'm open to having this logic here if it makes sense but I would think the ui would not need to control the construction of the board, but would take the board as a parameter and just be responsible for displaying what ever board it is given.  I think the responsibility of the shape of the board would belong to the game class, the ttt class would call to create the board then pass it to the ui to display.   BUT...   you are free to disagree.  let me know your thoughts

Reply:
I think there's a middle ground to be struck here, where the `Board` class exists but is only responsible for "setting up/drawing the board" and housing the empty board that is used to instantiate a playable board in TTT at the beginning of the game. This would mean the `Game` class would deal with the actual functionality to make moves and play the game. I guess I'm wondering if it has to be mutually exclusive or if both classes can still exist, but we reduce all of the excess responsibility I pushed into `Board` previously.

Reply:
IMO, `Board` class should deal with most things board related: methods that format the board (different than printing the board) and mark the board, logic for if the board is full, logic for if a spot is taken...absolutely both classes can still exist. Asking the question, whose responsibility should this be - what class should care about this functionality or this information? The `Game` class can still make moves and play the game as your main class, so to speak, but should it house all the information about the board, like if the board is full or if a spot is already taken? That seems like `Board` responsibility to me.

`UserInterface` can print things and pull things from the console, but it really shouldn't need to format anything. Just take things in and print things out.

Reply:
I said board class, I meant Game class.   you should rename the ruby file to game.rb